P6-Introduction-Table-Views/content.md

```diff
@@ -36,17 +36,17 @@ I would encourage you to always look at new objects under the various *Inspector
 You may have noticed that if you click on the `View Controller` and inspect under the *Identity Inspector*, it is using a custom class of **NotesViewController**. 
 You will be expanding on this class very soon.
  
-You will come back to this shortly. However, let's look at the other `Controller` in our Storyboard.
+You will come back to this shortly. However, let's first look at the other `Controller` in our Storyboard.
  
 ##Navigation Controller
  
-What is a *Navigation Controller* ?
+What is a *Navigation Controller*?
 
 The *Navigation Controller* manages a stack of view controllers. It provides a drill-down interface for hierarchical content. It often goes hand in hand with Table Views - 
-for example look at the *Photos* App. Click `Albums` to take you into a View Controller that presents the UITableView, click a row and it opens 
-a View Controller to display image thumbnails. Drill down again and you go into another View Controller that will preview the image.
+for example, look at the *Photos* App. Tapping on `Albums` takes you into a View Controller that presents the UITableView.  Tapping on a row opens 
```

Comment:
`` should ideally only refer to code, so here it should be **. I think @cocojoe would agree ;)